### PR TITLE
ADD MEDIA, HTML AND CSS FILES

### PR DESCRIPTION
**Pull request by Gdiazdiaz**
### TASK ACCOMPLISHED
- [x] Add a headline
- [x] Add a video
- [x] Add space to username, email and password
- [x] Add logo

### CODE REVIEW

**Comment by Gdiazdiaz** [comment]
Hi! You're doing a great job Glenda =) however there are some things we can improve
You need to add a .gitignore file and and node_modules inside.
You also need the linters for HTML&CSS
Also remember to be more informative in your commits regarding what changed in your project

**Comment by Gdiazdiaz** [comment]
`jhbjhbjhbjhbbjh`

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-
```
Comment (Gdiazdiaz) [inline README.md:1]: Your README file needs to be more descriptive

File: Style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+body{
+    background-color: darksalmon;
+}
```
Comment (Gdiazdiaz) [inline Style.css:3]: You are definitly missing some stuff, you can use css grid or flexbos to tidy up a little, use margin and padding to have a more clean space, and also use height and width to fix the logo and the video

File: index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    Pianos DiViot
+    <img id="header-img" src="white piano.jpg">
+    <nav id="nav-bar">
+        <button class="nav-link" onclick="">Concerts</button>
+        <button class="nav-link" onclick="">Types of pianos</button>
+        <button class="nav-link" onclick="">Studios</button>
+    </nav>
+</header>
+<body>
+    <video width="320" height="240" controls>
```
Comment (Gdiazdiaz) [inline index.html:20]: this will look more clean with a div for the video and it's images

File: README.md (hunk 1 above)
Comment (bienvenuushindi) [reply README.md:1]: [OPTIONAL] This comment will look great :100:  if you include an explanation, a "why", for this change. 
For example, You would  say 
**Your README file needs to be more descriptive. A well-crafted description allows you to show off your work to other developers as well as potential employers**

**Review by bienvenuushindi — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @Gdiazdiaz    👋, 

Your project is complete!  👏  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights 🎯 
:heavy_check_mark: Descriptive PR title & description.
:heavy_check_mark: Inline comments are present
:heavy_check_mark: Comments are clear and concise :clap: 

### Optional Suggestions
- [ ] You can include emojis and gifs to make your code review more human-friendly :alien:.
- [ ] Kindly Check inline comment  under this review

**_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**